### ISSUE TITLE
DatePicker keyboard navigation

### ISSUE THREAD

**Issue by mrjanuszm** [mkpaz/atlantafx#176]
Keyboard navigation does not work correctly in the DatePicker control.
It is not possible to select a day in the calendar using the arrow keys.
In the Modena theme, everything works correctly.

**Comment by mrjanuszm** [comment]
I investigated this more thoroughly, and it turns out that the cursor does work in the DatePicker.
However, the styling of the current day under the cursor makes the cursor invisible.
This can be fixed by applying the following styles to the DatePicker
/* The day indicated by the keyboard cursor */
.date-picker-popup > * > .day-cell:focused {
    -fx-background-color: #ffaa00; /* Any bright color, orange in this case */
    -fx-text-fill: black;          /* Text color of the day number */
}

/* Optional: when the same focused day is also hovered by the mouse */
.date-picker-popup > * > .day-cell:focused:hover {
    -fx-background-color: #e69900;
}
